src/app/legacy/psammead/psammead-story-promo-list/src/index.jsx

```diff
@@ -35,8 +29,6 @@ export const StoryPromoLiBase = styled.li`
   }
 `;
 
-StoryPromoLiBase.propTypes = promoListPropTypes;
-
 StoryPromoLiBase.defaultProps = {
```

Comment:
have these been missed?

Reply:
As above, same as the other Emotion styled components.